MeesGame/UI/Controls/EditorControls/ColorPicker.cs

```diff
@@ -22,14 +22,12 @@ class ColorPicker : UIComponent
             Color.Cyan,
             Color.Magenta,
             Color.Yellow,
-            Color.Brown,
-            Color.Gold
         };
 
         /// <summary>
         /// The colors from which the editor can choose.
         /// </summary>
-        private TextureButton[] colors = new TextureButton[8];
+        private TextureButton[] colors = new TextureButton[6];
```

Comment:
Don't use magic numbers. Use the amount of colors available if possible or a constant.